PA193_mnemonic_Slytherin/mnemonic.py

```diff
@@ -132,8 +201,6 @@ def verify(mnemonic: str, expected_seed: bytes, seed_password: str = '') -> bool
     :rtype: bool
     :return: True if provided phrase generates expected seed, False otherwise.
     """
-    if not __is_valid_mnemonic(mnemonic):
```

Comment:
Validation of mnemonic is removed here, and unchecked `mnemonic` is passed to `__generate_seed` which does not perform any validation at the moment. This approach could be fine, but this PR might sync with *dev* branch when #21 is merged so that `__generate_seed` actually does any checks.

However, as of ebaa9ed6a277b522167eda3c7be2e60f1644e0ed,  `__generate_seed` **does not validate** mnemonic. 

https://github.com/mvondracek/PA193_mnemonic_Slytherin/blob/04ab60aab39a998ace40d9c73cabaee988474e65/PA193_mnemonic_Slytherin/mnemonic.py#L24-L35

Reply:
Would be solved by #26.

Reply:
But that would be for a separate PR, don't worry. 😄